source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```
@@ -55,103 +96,127 @@ content: |
+           providerDetails.put("email", new BsonString("<GCP service account email>"));
+           providerDetails.put("privateKey", new BsonString("<GCP service account private key>"));
+           providerDetails.put("endpoint", new BsonString("<GCP authentication endpoint>"));
+
+           kmsProviders.put("gcp", providerDetails);
 
-           kmsProviders.put("aws", providerDetails);
      .. tab::
         :tabid: nodejs
 
         .. code-block:: javascript
 
+           // TODO: check correctness
            kmsProviders = {
-             aws: {
-               accessKeyId: '<IAM User Access Key ID>',
-               secretAccessKey: '<IAM User Secret Access Key>',
+             gcp: {
+               email: '<GCP service account email>',
+               privateKey: '<GCP service account private key>',
+               endpoint: '<GCP authentication endpoint>',
              }
            }
+
      .. tab::
         :tabid: python
 
         .. code-block:: python
 
-           kms_providers = {
-               "aws": {
-                   "accessKeyId": "<IAM User Access Key ID>",
-                   "secretAccessKey": "<IAM User Secret Access Key>"
+           kms_provider = {
+               "gcp": {
+                   "email": "<GCP service account email>",
+                   "privateKey": "<GCP service account private key>",
+                   "endpoint": "<GCP authentication endpoint>",
                }
            }
 ---
 title: Create a New Data Encryption Key
-ref: create-a-new-data-key
+ref: create-a-new-data-key-gcp
 content: |
-  The following diagram shows how the **customer master key** is created and
-  stored when using a KMS provider:
+  In this step, we generate a new **data encryption key** using the
+  **master key** in the remote KMS. The following diagram shows the
+  requests you make from the client application to create and store a
+  new data encryption key:
 
   .. image:: /figures/CSFLE_Data_Key_KMS.png
      :alt: Diagram that describes creating a data encryption key when using a KMS provider
 
-  You must generate a new **data encryption key** using the **master key**
-  in the remote KMS. The original data encryption key was encrypted by
-  your locally-managed master key.
+  Provide your client with the following information to access the master key:
+
+  .. list-table::
+     :header-rows: 1
+     :stub-columns: 1
 
-  Specify the AWS region and `Amazon Resource Number
-  <https://docs.aws.amazon.com/kms/latest/developerguide/viewing-keys.html#find-cmk-id-arn>`_
-  (ARN) of the new CMK in the CSFLE-enabled client settings. Use the client
-  to create a new data encryption key as follows:
+     * - Field
+       - Required
+       - Description
 
-  Once you have the required information, run the following code to
-  generate the new data encryption key:
+     * - projectId
+       - Yes
+       - The identifier for your project in which you created the key.
```

Fix: Parallelism, this should match the syntax of the other fields.
```suggestion
       - Identifier for your project in which you created the key.
```